docs/Using Fleet/Usage-statistics.md

```diff
@@ -25,6 +25,9 @@ Below is the JSON payload that is sent to Fleet Device Management Inc:
   "hostExpiryEnabled": true,
   "mdmWindowsEnabled": false,
   "liveQueryDisabled": false,
+  "aiFeaturesDisabled": false,
```

Comment:
### Dev note
Whether `server_settings.ai_features_disabled` is set to `true` in the config